docs/ReleaseNotes.md

```diff
@@ -1,6 +1,13 @@
 # Release Notes
 
-## Unreleased changes:
+# Important: .NET 4.0 support is currently **disabled**; if you need .NET 4.0, please stick with 1.2.1 while we keep investigating
```

Comment:
something seems off with the baselining here; what is the baseline for this PR? it seems to be showing relative to some very old version; is there any chance of rebasing against current?